test/tweet_test.rb

```diff
@@ -34,6 +34,13 @@ def test_that_a_tweet_cant_be_found_works
     end
   end
 
+  def test_that_a_tweet_can_have_no_media
+    tweet = Birdsong::Tweet.lookup("20").first
+    assert_equal tweet.id, "20"
+    assert_not_nil tweet.image_file_names
+    assert_equal 0, tweet.image_file_names.count
```

Comment:
add checks for videos as well, since we want to make sure *no* media was collected.